In [24]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import os

import dhs_util
from dhs_util import *

os.chdir('C:/Users/lukeh/OneDrive/Documents/College/Topics in Data Science/Unit 1')
df = pd.read_csv('dhs_service_records_synthesized_final.csv')
df = dhs_preprocessing(df)
df, service_map = add_service_label(df)
df = add_age_bin(df)

recipient = get_recipient_attribute(df)

In [25]:
df_1 = df.query("service == 'S14'")
df_2 = df.query("service != 'S14'")


In [40]:
age_lower = 24
age_upper = 55
id_cohort_age = df.query("age > @age_lower and age <= @age_upper").id.unique()
recipient_1 = get_recipient_attribute(df_1)
recipient_2 = get_recipient_attribute(df_2)

In [38]:
id_white_df_1 = df_1.query("race == '1~White'")
id_white_df_2 = df_2.query("race == '1~White'")
id_not_white_df_1 = df_1.query("race != '1~White'")
id_not_white_df_1 = id_not_white_df_1.query("race != '99~Unknown'")
id_not_white_df_2 = df_2.query("race != '1~White'")
id_not_white_df_2 = id_not_white_df_2.query("race != '99~Unknown'")

In [ ]:
id_white_df_1.nunique()

In [ ]:
id_white_df_2.nunique()

In [47]:
perc_white_using_S14 = 36801/ (36801+334635)
perc_white_using_S14

0.09907763383193875

In [ ]:
id_not_white_df_1

In [ ]:
id_not_white_df_2

In [48]:
perc_not_white_using_S14 = 17417/(17417+177875)
perc_not_white_using_S14

0.08918440079470741

### Conclusion
We found that white residents of this country were more likely to use Mental Helth services (S14) at 9.9% compared to the 8.9% of non white residents white recieved the same mental heath services
This result could be for many different reasons but one could be that the county makes these services less accessable in areas that primarly have non white residents. 